### ISSUE TITLE
read instruction fails for files outside Naproche-SAD/examples

### ISSUE THREAD

**Issue by McEarl** [Naproche/Naproche-SAD#4]
The `read` instruction cannot handle files which are stored outside the _Naproche-SAD/examples_ directory.

Example:

`[read /home/user/forthel/test.ftl]`

produces the error message

`[Parser] (file "examples//home/user/forthel/test.ftl") does not exist`.

**Comment by anfelor** [comment]
Yes, we know of this behavior and will probably be fixed in the next release. In the meantime you can either put all your files in the examples directory or create a symlink to another directory.

**Comment by anfelor** [comment]
Thanks for opening all of these! It really helps to have some user reports!
